index.bs

```diff
@@ -1594,19 +1594,39 @@ Constructors</h4>
 
 			1. Set a {{[[control thread state]]}} to <code>suspended</code> on the {{AudioContext}}.
 
-			2. Set a {{[[rendering thread state]]}} to <code>suspended</code> on the {{AudioContext}}.
+			1. Set a {{[[rendering thread state]]}} to <code>suspended</code> on the {{AudioContext}}.
 
-			3. Let <dfn attribute for="AudioContext">[[pending resume promises]]</dfn> be a
+			1. Let <var>messageChannel</var> be a new {{MessageChannel}}.
```

Comment:
Same here. Using `|messageChannel|` should be simpler and shorter.

Reply:
I meant using the pipe to denote algorithm variables.

Reply:
Fixed, and added the necessary markup.